osquery/sql/virtual_table.cpp

```diff
@@ -979,6 +985,12 @@ static int xFilter(sqlite3_vtab_cursor* pVtabCursor,
     }
   }
 
+  if (content->colsUsedBitsets.size() > 0) {
```

Comment:
This is almost nothing, but all containers has a method `.empty()`. Which is more expressive, imho :)